Makefile

```diff
@@ -39,6 +39,17 @@ install-extra: extra
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) -m755 $(EXTRA_TARGETS) $(DESTDIR)$(PREFIX)/bin
 
+uninstall:
+	rm -f $(DESTDIR)$(PREFIX)/bin/f3write
+	rm -f $(DESTDIR)$(PREFIX)/bin/f3read
+	rm -f $(DESTDIR)$(PREFIX)/share/man/man1/f3read.1
+	rm -f $(DESTDIR)$(PREFIX)/share/man/man1/f3write.1
+
+uninstall-extra:
+	rm -f $(DESTDIR)$(PREFIX)/bin/f3probe
+	rm -f $(DESTDIR)$(PREFIX)/bin/f3brew
+	rm -f $(DESTDIR)$(PREFIX)/bin/f3fix
```

Comment:
Would it be possible to use `$(TARGETS)` in the label `uninstall` and `$(EXTRA_TARGETS)` in `uninstall-extra` to avoid repeating the names of the binaries?

Reply:
Done